remsfal-service/src/main/java/de/remsfal/service/boundary/BuildingResource.java

```diff
@@ -0,0 +1,75 @@
+package de.remsfal.service.boundary;
+
+import de.remsfal.core.api.project.BuildingEndpoint;
+import de.remsfal.core.json.project.BuildingJson;
+import de.remsfal.core.model.project.BuildingModel;
+import de.remsfal.service.control.BuildingController;
+import jakarta.enterprise.context.RequestScoped;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.WebApplicationException;
+import jakarta.ws.rs.container.ResourceContext;
+import jakarta.ws.rs.core.Context;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import jakarta.ws.rs.core.UriInfo;
+
+import java.net.URI;
+
+@RequestScoped
+public class BuildingResource implements BuildingEndpoint {
+    @Context
+    UriInfo uri;
+
+    @Context
+    ResourceContext resourceContext;
+
+    @Inject
+    BuildingController controller;
+
+
+    @Override
+    public Response createBuilding(String projectId, String propertyId, BuildingJson building) {
+        try {
```

Comment:
Please check the user privileges for the project referenced by project id. 
This is the top one security risk listed in OWASP top 10 API, https://owasp.org/API-Security/editions/2023/en/0x11-t10/